src/Mixin.php

```diff
@@ -12,8 +8,10 @@
 use Throwable;
 
 /**
- * This trait aids static analysis tooling in introspecting assertion magic methods.
+ * This trait provides nurllOr* and all* variants of assertion base methods.
  * Do not use this trait directly: it will change, and is not designed for reuse.
+ *
+ * @internal
```

Comment:
This is not directly related to this pr, but I think the trait should be marked with `@internal`.

Reply:
@Ocramius since you wrote the sentence above: is there a reason you didn't add the `@internal` tag?

Reply:
Yes: if it was `@internal`, all methods on this trait would also be `@internal`.

Reply:
Are you sure? 
phpstorm and psalm handle it differently: `@internal` on trait level does not mark the methods as internal, if the trait is used in a class of the same package (and unless the methods are marked as internal themselves).

psalm: https://psalm.dev/r/7a6e99b588

Reply:
hmm actually psalm seems to completely ignore the internal tag on trait level, even if the trait is used in another namespace: https://psalm.dev/r/2db239456c

but phpstorm is handling it like I described above.

Since the correct behavior is not clear and since it is not related to the main purpose of this pr, I think I should revert the line.